javascript-functions/main.js

```diff
@@ -0,0 +1,33 @@
+function convertMinutesToSeconds(minutes) {
+  return (minutes * 60);
+}
+
+var seconds = convertMinutesToSeconds(6);
+console.log('6 minutes is equivalent to: ' + seconds + ' seconds');
+
+function greet(name) {
+  console.log('Hello ' + name);
```

Comment:
You should `return` this value, and log the return when you call it. In general, functions should always `return` the result of its work, so it can be used outside the function.